### DISCUSSION TITLE
Add album section

### DISCUSSION THREAD

**Discussion by leosamuele221** [Radiokot/photoprism-android-client#34] [Ideas]
👉🏻 [Start discussing an idea](https://github.com/Radiokot/photoprism-android-client/discussions/new?category=ideas)

Add a page to view albums and a toggle in settings to use folders as albums

**Comment by ippocratis** [comment]
Suggestions and ideas is probably more suited in the discussion section
There is  already one open with a similar request
https://github.com/Radiokot/photoprism-android-client/discussions/20
But you already are linking the new suggestion
I'm confused

**Comment by leosamuele221** [comment]
I think that link is set automatically, I forgot to delete it.
I just saw the discussion #20 . 
I would add this feature myself but I don't have much time and I'm not familiar with kotlin

**Reply by Radiokot** [reply]
Hi Samuele 👋🏻

Thank you for checking out the app and describing your ideas. You're right, it would be useful to be able to view content of particular albums and folders in the app. Currently, it can be done by making a bookmark with an "album:XXX" or "folder:"XXX/YYY/ZZZ" query. However, this requires you to know the album/folder exact name, so it is not super convenient. 

Soon I'll add the ability to filter the content by albums and folders with a simple selection. Subscribe to the repo to get notified on new releases 👀

**Reply by ippocratis** [reply]
@leosamuele221 thanks for albums
Very usefull
